bin/generate-fasta.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+"""
+convert peptide metadata to fasta format.
+"""
+
+import pandas as pd
+import sys
+
+def trim_index(sequence):
+    return "".join([nt for nt in sequence if nt.isupper()])
+
+fasta_fp = open(f"{sys.argv[1]}.fasta", "w")
```

Comment:
It's always nice to use a closure for file handles in Python to avoid leaving them open by mistake.

e.g.
```
with open(f"{sys.argv[1]}.fasta", "w") as fasta_fp:
    fasta_fp.write()
```